### ISSUE TITLE
database update status

### ISSUE THREAD

**Issue by DanielMedic** [KechrisLab/multiMiR#31]
Here's the current status of all databases, from the listing at <http://multimir.ucdenver.edu>.  Most of them need updating, obviously.  Once I'm added to the project as a contributor, I'll self-assign this issue, and get access from @kechrisk or @smahaffey to update the databases on the server.

Databases that appear to be up to date:

- miRanda has a current date of August, 2010, which is the same as the latest on the linked database.
- PITA is currently version 6, same as the latest on the link.
- PhenomiR has a current date of February 15th, 2011.  This appears to be up to date.

Databases that need updating:

- miRTarBase is currently version 6.1.  The latest version is 7.0.
- TarBase is currently version 6, latest is 7.0.
- miRDB is currently version 5, latest is 6 <http://mirdb.org/download.html>.
- TargetScan is currently version 7.1, latest is 7.2.  The link on <http://multimir.ucdenver.edu> points at <http://www.targetscan.org/cgi-bin/targetscan/data_download.cgi?db=vert_61>, and should probably be updated just to point at <http://www.targetscan.org/>.

Databases with uncertain status:

- Pharmaco-miR doesn't have a version number, so there's no way to tell (other than `diff`ing the files, I guess) if it's up to date.  The link on <http://multimir.ucdenver.edu> should probably point to the project home page at <http://www.pharmaco-mir.org>.
- DIANA-microT-CDS is currently version 5.  Attempting to contact the linked page <http://diana.cslab.ece.ntua.gr/data/public/> gets a 404 error.  Is <http://diana.imis.athena-innovation.gr/DianaTools/index.php?r=microT_CDS/index> the new site?  If so, it looks like the version hasn't changed, but the link on <http://multimir.ucdenver.edu> should be changed.
- The link for PicTar <http://dorina.mdc-berlin.de/> now leads to a tool called doRiNA.  Without digging too deep, this implies to me that the tool and maybe the data format have changed considerably <https://www.ncbi.nlm.nih.gov/pubmed/25416797>.  All references to PicTar itself, e.g. <https://pictar.mdc-berlin.de> look very old.

Databases with broken links:

- miRecords is currently version 4.  I am unable to contact any page on the site <http://mirecords.biolead.org>, nor to any page which comes up with a Google search for "miRecords," which does not fill me with confidence.
- ElMMo is currently version 5.  The original link <http://www.mirz.unibas.ch/miRNAtargetPredictionBulk.php> gives a 404.  Googling for "ElMMo microRNA" directs me to <https://omictools.com/elmmo-tool> with the message "This tool is not available anymore."
- MicroCosm is currently version 5.  It appears to be no longer maintained at the linked site <http://www.ebi.ac.uk/enright-srv/microcosm/cgi-bin/targets/v5/download.pl>.  Googling turns up no useful links.
- The current version of miR2Disease is 7.1, dated June 2016.  I am unable to open the link <http://www.mir2disease.org/> or find any recent references on Google.

The first two categories are easy enough to deal with.  The third ("uncertain status") may take some more work.  And I'm honestly not sure what to say about the fourth ("broken links").  Do we want to leave them as they are?  If so, should we warn the user, and maybe provide an option _not_ to use anything that isn't known to be up to date?  Something to discuss.

**Comment by umitozorhan** [comment]
Dear Daniel,

Thank you for informing us about the issue. As far as I know, **Diana microT CDS 5** now located in "diana.imis.athena-innovation.gr/DianaTools/index.php?r=microT_CDS/index". For reaching **miRecords**, I found this website "http://c1.accurascience.com/miRecords/" however this looks like a v4.0 also they give some information about that page in this link "https://www.lcsciences.com/discovery/mirecords-a-new-integrated-resource-for-microrna-target-interactions/". 

 Thus implementing **miRbase v22.1** could be a good idea. But I do not know, it could be possible or not. (For example, an unknown miRNA written in the search, the system could simply say this miRNA does not exist yet. **miRbase v22.1** "http://www.mirbase.org"

Thusi it is worth noting to implement new databases to improve coverage such as **TarPmiR** "http://hulab.ucf.edu/research/projects/miRNA/TarPmiR/", **SM2miR** "http://210.46.85.180:8080/sm2mir/index.jsp", **PhenomiR 2.0** "http://mips.helmholtz-muenchen.de/phenomir/", **EpimiR** "http://www.jianglab.cn/EpimiR/", **TSmiR** "http://lcbb.swjtu.edu.cn/TSmiR/",  **Imota** "https://ccb-web.cs.uni-saarland.de/imota/sources/", **Exocarta** for exosomal miRNA "http://exocarta.org/". Also, **HMDD v3.0** "http://www.cuilab.cn/hmdd", **miR2disease**(i don't know if it still exists), **starbase v3.0** "http://starbase.sysu.edu.cn", **Exo-miRExplorer** (Only chinese access might be talk with to database owners) could be good for increasing the coverage further.

**Comment by DanielMedic** [comment]
Okay, thanks.  Including the new databases sounds like a good idea, although I'd like to concentrate on getting everything we currently have as up to date as possible.  But after that, yes, I think we should try adding them one at a time, with thorough testing on each.

**Comment by DanielMedic** [comment]
Tarbase has actually been updated to version 8, but that requires agreement with their license terms <http://carolina.imis.athena-innovation.gr/diana_tools/web/index.php?r=tarbasev8%2Fdownloaddataform>.  Our current version is v.6, and v.7 is available for download without agreeing to the terms.  Do we want to go to v.8, or just stick with v.7?  We can't really guarantee that people will only use multiMiR "for scientific non-profit and non-commercial use only."  For now I'm going to use v.7 and we can think about going to v.8 later.

**Comment by smahaffey** [comment]
Tarbase is the one we tried repeatedly to contact them about a license and we couldn't ever get a response.  I think for now v7 works. It's the version we kept contacting them to use.

On Jul 6, 2019 12:53 PM, DanielMedic <notifications@github.com> wrote:

Tarbase has actually been updated to version 8, but that requires agreement with their license terms http://carolina.imis.athena-innovation.gr/diana_tools/web/index.php?r=tarbasev8%2Fdownloaddataform. Our current version is v.6, and v.7 is available for download without agreeing to the terms. Do we want to go to v.8, or just stick with v.7? We can't really guarantee that people will only use multiMiR "for scientific non-profit and non-commercial use only." For now I'm going to use v.7 and we can think about going to v.8 later.
